mis_builder_cash_flow/report/mis_cash_flow.py

```diff
@@ -77,15 +75,16 @@ def init(self):
                 aml.full_reconcile_id as full_reconcile_id,
                 aml.partner_id as partner_id,
                 aml.company_id as company_id,
-                aml.user_type_id as user_type_id,
+                aa.user_type_id as user_type_id,
```

Comment:
Perhaps this change can add a little of overhead because of an additional join?
Now that we have `account_internal_type` on aml. perhaps we can add this field instead of `user_type_id` and then on all the expressions (like [this one](https://github.com/OCA/account-financial-reporting/blob/4d15be2e1e592e978cee55c713a663ebf3d8ab15/mis_builder_cash_flow/data/mis_report.xml#L39)) use just `account_internal_type` instead of `account_id.internal_type` (what could improve performance).

What do you think?

Reply:
I think there is no overhead. But I will try what you said.

Reply:
The overhead is already there if you have `account_account aa` in the `JOIN` part. If you can avoid it because there's no need of any of its fields, then you boost up performance.

Reply:
Well, I just take out this field, as it was not used anywhere. Now, there is an error in account_financial_report/tests/test_vat_report.py, but seems unrelated, right?

Reply:
Great, thanks @MiquelRForgeFlow !
I still believe that adding `account_internal_type` and replacing `account_id.internal_type` for `account_internal_type` could help in performance but that could be achieved on a new PR.

Thanks!

Reply:
Done :)